packages/web-components/src/components/ic-text-field/ic-text-field.tsx

```diff
@@ -158,13 +161,23 @@ export class TextField {
   /**
    * The maximum number of characters that can be entered in the field.
    */
+  @Prop() maxCharacters: number = 0;
+
+  /**
+   * The label for maximum number of characters that can be entered in the field.
```

Comment:
This comment for MaxLength doesn't seem to line up - how can a label be a number?